_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/files/tables/resources.csv

```diff
@@ -0,0 +1,5 @@
+1, instances
```

Comment:
Agree with this.  One of the nice things about having our ansible scripts in the ciao repo is that we can avoid this sort of duplication.  We should take advantage of this.